tcpdf.php

```diff
@@ -18922,13 +18922,26 @@ protected function openHTMLTagHandler($dom, $key, $cell) {
 				$prevlinewidth = $this->GetLineWidth();
 				$this->SetLineWidth($hrHeight);
 
-				$lineStyle = array(
-					'color' => $tag['fgcolor'],
-					'cap'   => $tag['style']['cap'],
-					'join'  => $tag['style']['join'],
-					'dash'  => $tag['style']['dash'],
-					'phase' => $tag['style']['phase'],
-				);
+				$lineStyle = array();
+
+				if (isset($tag['fgcolor'])) {
+					$lineStyle['color'] = $tag['fgcolor'];
+				}
+
+				if (isset($tag['style']) && is_array($tag['style'])) {
+					if (isset($tag['style']['cap'])) {
+						$lineStyle['cap'] = $tag['style']['cap'];
+					}
+					if (isset($tag['style']['join'])) {
+						$lineStyle['join'] = $tag['style']['join'];
+					}
+					if (isset($tag['style']['dash'])) {
+						$lineStyle['dash'] = $tag['style']['dash'];
+					}
+					if (isset($tag['style']['phase'])) {
+						$lineStyle['phase'] = $tag['style']['phase'];
+					}
```

Comment:
Wouldn't be better just to use much shorter PHP's null coalescing operator syntax?
```php
				$lineStyle = array(
					'color' => $tag['fgcolor'],
					'cap'   => $tag['style']['cap'] ?? null,
					'join'  => $tag['style']['join'] ?? null,
					'dash'  => $tag['style']['dash'] ?? null,
					'phase' => $tag['style']['phase'] ?? null,
				);
```

Reply:
I belive this is PHP 7 only...

Reply:
You are right.